ethr.go

```diff
@@ -114,6 +116,12 @@ func main() {
 		ipVer = ethrIPv6
 	}
 
+	//Default latency test to 1KB if length is not specified
+	switch *bufLenStr {
```

Comment:
Would it not be simpler & neater to set the default in the flag line? i.e. at line 45, 
```
bufLen := flag.String("l", "1B", "buffer length")
```

and then override it later if our test is of a latency type?

Reply:
@boyan-soubachov That would break regressions with tests that are not latency. How would you know when someone didnt specify a value for a bandwidth test? Let's say someone wanted to run a bandwidth test ```ethr -c client -t b ```. For that we would want to default it to use 16KB since its not a latency test. How would we know if the user passed in "1B" or passed in no argument? At that point, you cant tell. The problem is this flag is used across multiple types of tests. The easiest way is to default it to empty string to begin with and then populate it to some default based on test after the values have been read from the command line